index.js

```diff
@@ -349,7 +357,7 @@ async function go() {
             console.log('Pushing to remote "origin"...');
             await git(clonePath, `push ${repoFullUrl} refs/heads/${branchName}`);
 
-            if (await findGithubFile('PULL_REQUEST_TEMPLATE') && !quiet) {
+            if (!quiet && !createPR) {
```

Comment:
If the user sets the `--no-pr` and `--quiet` flags, `quiet === true` so a PR will be created. That's not the behaviour I would expect.

Reply:
Oops...